client/cgo/cgotest.go

```diff
@@ -0,0 +1,222 @@
+package main
+
+/*
+int testVerifyVrf(unsigned char *pk, int pkSize,
+    unsigned char *m, int mSize,
+    unsigned char *index, int indexSize,
+    unsigned char *proof, int proofSize) {
+    return cgoVerifyVrf(pk, pkSize, m, mSize, index, indexSize, proof, proofSize);
+}
+
+int testVerifySignature(
+    unsigned char *pk, int pkSize,
+    unsigned char *m, int mSize,
+    unsigned char *sig, int sigSize) {
+    return cgoVerifySignature(pk, pkSize, m, mSize, sig, sigSize);
+}
+
+int testHashChainInvalidSize() {
```

Comment:
Well, no, having a test for STR linear history audit is good.  And we need a test covering `cgoVerifyHashChain`.  Just seems incomplete.